frontend/src/lib/components/apps/components/helpers/RunnableWrapper.svelte

```diff
@@ -97,6 +98,9 @@
 			sendUserToast(doOnSuccess.configuration.sendToast.message)
 		}
 	}
+
+	$: currentAutoRefresh =
+		componentInput?.type === 'runnable' ? componentInput.autoRefresh : autoRefresh
```

Comment:
can this not be handled by the RunnableComponent directly ?

Also you can just inline this since it's only used once